### PR TITLE
Adding goat counter and bumping docs theme version

### PR DESCRIPTION
**Pull request by travishathaway**
### Description

xref: https://github.com/conda/conda-docs/issues/905

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [ ] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-conda-sphinx-theme==0.1.1
+conda-sphinx-theme==0.1.3
```
Comment (kenodegard) [inline docs/requirements.txt:1]: In conda we don't pin at all, should that change?

https://github.com/conda/conda/blob/d891d9f59f193a3e28bf28c959f41b08e304f31c/docs/requirements.txt#L1

**Review by jezdez — CHANGES_REQUESTED** [review]

File: docs/source/_templates/layout.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{% extends "!layout.html" %}
+
+{% block footer %}
+{{ super() }}
+
+{# Only embed analytics on production #}
+{% if use_analytics %}
+<script data-goatcounter="https://docs-conda-io.goatcounter.com/count"
+        async src="//gc.zgo.at/count.js"></script>
```
[the remark this answers is not in the document]
Comment (travishathaway) [reply docs/source/_templates/layout.html:9]: Good idea. I had forgotten that we did something similar for conda-dot-org.

File: docs/source/_templates/layout.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% extends "!layout.html" %}
+
+{% block footer %}
+{{ super() }}
+
+{# Only embed analytics on production #}
+{% if use_analytics %}
+<script data-goatcounter="https://docs-conda-io.goatcounter.com/count"
+        async src="_static/js/count.js"></script>
```
Comment (jezdez) [inline docs/source/_templates/layout.html:9]: I think that's not needed anymore?

Comment (travishathaway) [reply docs/source/_templates/layout.html:9]: oops

Comment (travishathaway) [reply docs/source/_templates/layout.html:9]: removing

**Review by jezdez — APPROVED** [review]

**Pull request merged**